src/api/java/com/minecolonies/api/colony/buildings/modules/ICraftingBuildingModule.java

```diff
@@ -37,6 +38,13 @@
  */
 public interface ICraftingBuildingModule extends IBuildingModule
 {
+    public enum CrafingType
+    {
+        CRAFTING,
+        SMELTING,
+        LARGE,
```

Comment:
It's because we got 2x2 and 3x3 recipes, don't we need them inherently separate for that? I mean we did have two methods for that before

Reply:
Yeah, but it's a modifier for the crafting type, it's not an entirely separate type.

Reply:
Could be a separate method entirely that only applies to "normal crafting" or also be it's own type. We could relatively easy adjust this to use the RecipeType and create our own RecipeType for Brewing and a separate LARGE type just for simple use.

Reply:
Definitely no LARGE type, no.  Do it the way I did it in my PR.  In fact, just merge my PR and then I might be able to help making this compatible with it. 😁

Reply:
I'm actually somewhat tempted to try making a PR for Forge to introduce a proper brewing recipe type.  Though since it's a fairly big API change the chances of it landing before 1.19 are fairly slim, I'd imagine, so that doesn't really help us.